api/lib/reducers/cosmosV0-reducers.js

```diff
@@ -358,7 +358,7 @@ function gasPriceReducer(gasPrice, coinLookup) {
   const denom = denomLookup(coinLookup, gasPrice.denom)
   return {
     denom: denom,
-    price: BigNumber(gasPrice.price).div(1000000) // Danger: this might not be the case for all future tokens
+    price: BigNumber(gasPrice.gasPrice).div(1000000) // Danger: this might not be the case for all future tokens
```

Comment:
`gasPrice.gasPrice` is correct?

Reply:
yes, it is called like that in DB, you can check